packages/mobile/scripts/run_e2e.sh

```diff
@@ -155,11 +155,6 @@ if [ $PLATFORM = "android" ]; then
 
     startPackager
 
-    # Kill all emulators if present
-    killall -9 qemu-system-x86_64 || true
```

Comment:
does removing this help to improve test stability, or is it just not needed? i was checking the history of this file and i see that we've had some code block to kill emulators that are alive at the time of running this script for a long time. github actions automatically cancels active jobs if new changes are pushed to the branch, so doesn't removing this block introduce problems?

Reply:
Ended up killing the emulators on Android in a different way:
```
NUM_DEVICES=`adb devices -l | grep emulator- | wc -l` || echo "0" | bc
if [ $NUM_DEVICES -ge 1 ]; then
  echo "$NUM_DEVICES emulator(s) already running or attached, removing"
  adb devices | grep emulator | cut -f1 | while read line; do adb -s $line emu kill; done
fi
 ```